app/controllers/buyers/accounts_controller.rb

```diff
@@ -71,12 +71,12 @@ def destroy
 
   def toggle_monthly_charging
     account.settings.toggle!(:monthly_charging_enabled)
-    redirect_to(:back)
+    redirect_back(fallback_location: redirection_path)
```

Comment:
is `:back` and `redirection_path` one and the same? Are these operations only done from `admin_buyers_account_path(account)`?

Reply:
redirect_to(:back) == redirect_back without fallback

And it generated so many issues that Rails decided to make it better with a fallback localtion